frontend/src/app/modules/members/members-autocompleter.component.ts

```diff
@@ -5,6 +5,7 @@ import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 import {HttpClient, HttpParams} from "@angular/common/http";
 import {Component} from "@angular/core";
 import {URLParamsEncoder} from "core-app/modules/hal/services/url-params-encoder";
+import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 
 export const membersAutocompleterSelector = 'members-autocompleter';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/29824e8f26ab94f31561587e9701a6895a7394ca/frontend/src/app/modules/members/members-autocompleter.component.ts#L11"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>